documentation/src/docs/pravega-concepts.md

```diff
@@ -11,362 +11,250 @@ You may obtain a copy of the License at
 
 The number of Stream Segments in a Stream determines the upper bound of
-parallelism of readers within a ReaderGroup – the more Stream Segments, the more
-separate, parallel sets of Readers we can have consuming the Stream. In the
-above figure, Stream1 has 4 Stream Segments.  That means that the largest
-effective Reader Group would contain 4 Readers.  Reader Group named "B" in the
-above figure is not quite optimal.  If one more Reader was added to the
-ReaderGroup, each Reader would have 1 Stream Segment to process, maximizing read
-parallelism.  However, the number of Readers in the ReaderGroup increases beyond
-4, at least one of the Readers will not be assigned a Stream Segment.
-
-If Stream1 in the figure above experienced a Scale-Down event, reducing the
-number of Stream Segments to 3, then Reader Group B as depicted would have an
+parallelism of readers within a Reader Group. If there are more Stream Segments, different Reader Groups and many parallel sets of Readers can effectively consume the Stream. In the
+above figure, **Stream 1** has four Stream Segments. The largest effective Reader Group would contain four Readers. **Reader Group B** in the above figure is not quite optimal. If one more Reader was added to the Reader Group, each Reader would have one Stream Segment to process, and maximizes the read
+parallelism. But, if the number of Readers in the Reader Group increases beyond four, there arises a possibility that at least one of the Readers will have unassigned Stream Segment.
+
+
+If **Stream 1** in the figure above experienced a **Scale-Down** event, by reducing the
+number of Stream Segments to three, then the **Reader Group B**  will have an
 ideal number of Readers.
 
-With the AutoScaling feature, Pravega developers don't have to configure their
-Streams with a fixed, pre-determined number of Stream Segments – Pravega can
-dynamically determine the right number.  With this feature, Pravega Streams can
-grow and shrink to match the behavior of the data input.  The size of any Stream
-is limited only by the total storage capacity made available to the Pravega
-cluster; if you need bigger streams, simply add more storage to your cluster.
+The number of Stream Segments can be varied dynamically by using the Pravega's auto scaling feature as we discussed in the [Auto Scaling](#elastic-streams-auto-scaling) section. The size of any Stream is determined by the storage capacity of the Pravega cluster. More Streams can be obtained by increasing the storage of the Pravega cluster.
```

Comment:
We don't need to talk again about the size of the Stream, we could simplify by deleting `The size of any Stream..` until ` Pravega cluster.`.